Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN /output/install-from-bindep && rm -rf /output/wheels
 COPY ansible-runner/context/app /runner
 
 # Pull in Quay dependencies
-FROM registry.redhat.io/quay/quay-rhel8:v3.6.1 as quay
+FROM registry.redhat.io/quay/quay-rhel8:v3.6.2 as quay
```

Comment:
Is there a way for us to have a single source of truth for the image versions?

Reply:
Fixed. Everything is set in `.env`